From 51889e560ffd85af0e6d94378dd58543ee839a93 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Tue, 23 Apr 2019 12:38:02 +0300
Subject: [PATCH 200/386] arm64: dts: a7k: move sfp node from setup level dts
 to board level dtsi

Cosmetic change.

Change-Id: I2478c131f03578078c1428375396f32be4ed98eb
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/7975
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-db-D.dts | 4 ----
 arch/arm64/boot/dts/marvell/armada-7040-db-E.dts | 4 ----
 arch/arm64/boot/dts/marvell/armada-7040-db.dts   | 4 ----
 arch/arm64/boot/dts/marvell/armada-7040-db.dtsi  | 4 ++++
 4 files changed, 4 insertions(+), 12 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
index c57a1e6193eb..5deb097f8b99 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
@@ -20,10 +20,6 @@
 		ethernet0 = &cp0_eth0;
 		ethernet1 = &cp0_eth2;
 	};
-
-	sfp_eth0: sfp-eth0 {
-		compatible = "sff,sfp";
-	};
 };
 
 &uart0 {
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts
index 9a214dc773a7..bbdf90a107e4 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts
@@ -20,10 +20,6 @@
 		ethernet0 = &cp0_eth0;
 		ethernet1 = &cp0_eth1;
 	};
-
-	sfp_eth0: sfp-eth0 {
-		compatible = "sff,sfp";
-	};
 };
 
 &uart0 {
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db.dts b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
index 3794b642826f..008e15edc262 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
@@ -23,10 +23,6 @@
 		ethernet1 = &cp0_eth1;
 		ethernet2 = &cp0_eth2;
 	};
-
-	sfp_eth0: sfp-eth0 {
-		compatible = "sff,sfp";
-	};
 };
 
 &i2c0 {
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi b/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi
index d5ff6ce12f46..30d78d85cb7f 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi
@@ -75,6 +75,10 @@
 		enable-active-high;
 		gpio = <&expander0 5 GPIO_ACTIVE_HIGH>;
 	};
+
+	sfp_eth0: sfp-eth0 {
+		compatible = "sff,sfp";
+	};
 };
 
 &uart0 {
-- 
2.17.1

